thinc/backends/cupy_ops.py

```diff
@@ -108,15 +108,15 @@ def frobenius_norm(X):
         gradient *= cupy.minimum(threshold, grad_norm) / grad_norm
         return gradient
 
-    def seq2col(self, seq, nW):
+    def seq2col(self, seq, nW, lens=None):
```

Comment:
The user-facing keyword needs to be `lengths` rather than `lens` for consistency (the variable is named `lengths` on `Ragged`) 

```suggestion
    def seq2col(self, seq, nW, *, lengths=None):
```